app/assets/javascripts/initializers/initializeBodyData.js

```diff
@@ -20,15 +27,15 @@ function fetchBaseData() {
       meta.name = 'csrf-param';
       meta.content = json.param;
       document.getElementsByTagName('head')[0].appendChild(meta);
-      var meta = document.createElement('meta');
+      var meta = document.createElement('meta'); // eslint-disable-line no-redeclare
       meta.name = 'csrf-token';
       meta.content = json.token;
       document.getElementsByTagName('head')[0].appendChild(meta);
       document.getElementsByTagName('body')[0].dataset.loaded = 'true';
-      if (checkUserLoggedIn()) {
+      if (window.checkUserLoggedIn()) {
```

Comment:
you can tell eslint that `checkUserLoggedIn` by adding `/* global checkUserLoggedIn */` at the beginning of the file

Reply:
Sounds good! I’ll make those changes sometime today!